scripts/prepare.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs')
```

Comment:
Missing prettier?

Reply:
I'm pretty sure it's there working but I'll check again.

what made you think it's missing? 🤔👀👀

Reply:
code doesn't look like rest of codebase (big 8 space tabs)

Reply:
I'll check 👌.

Reply:
In the end, it was my default prettier settings. I added an rc file. Good catch, thanks!